toolbox/strings.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import re
+import typing as t
+
+__all__: t.Sequence[str] = ("format_dt", "utcnow", "is_url", "is_invite")
+
+VALID_TIMESTAMP_STYLES: t.Sequence[str] = ("t", "T", "d", "D", "f", "F", "R")
+
+
+LINK_REGEX = re.compile(
+    r"https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()!@:%_\+.~#?&\/\/=]*)"
+)
+INVITE_REGEX = re.compile(r"(?:https?://)?discord(?:app)?\.(?:com/invite|gg)/[a-zA-Z0-9]+/?")
+
+
+def format_dt(time: datetime.datetime, style: t.Optional[str] = None) -> str:
+    """
+    Convert a datetime into a Discord timestamp.
+    For styling see this link: https://discord.com/developers/docs/reference#message-formatting-timestamp-styles
+
+    Parameters
+    ----------
+    time : datetime.datetime
+        The datetime to convert.
+    style : str, optional
+        The style to use for the timestamp, by default None.
+
+    Returns
+    -------
+    str
+        The formatted timestamp.
+    """
+
+    if style and style not in VALID_TIMESTAMP_STYLES:
+        raise ValueError(f"Invalid style passed. Valid styles: {' '.join(VALID_TIMESTAMP_STYLES)}")
+
+    if style:
+        return f"<t:{int(time.timestamp())}:{style}>"
+
+    return f"<t:{int(time.timestamp())}>"
+
+
+def utcnow() -> datetime.datetime:
```

Comment:
This definitely does not relate to strings

Reply:
Honestly not quite sure where to put it.

Reply:
Can do `time.py` or `utcnow.py`

Reply:
I agree, though I would recommend avoid using time.py as the filename due to potential namespace conflicts.